app/src/main/java/com/sma2/sma2/FeatureExtraction/Movement/WalkingFeatureActivity.java

```diff
@@ -0,0 +1,230 @@
+package com.sma2.sma2.FeatureExtraction.Movement;
```

Comment:
Please in this file use the radar manager that already exists

Reply:
Done